cloud/deployment/fromfile/fromfile.go

```diff
@@ -411,36 +625,36 @@ func deploymentExists(existingDeployments []astro.Deployment, deploymentNameToCr
 
 // deploymentFromName takes existingDeployments and deploymentName as its arguments.
 // It returns the existing deployment that matches deploymentName.
-func deploymentFromName(existingDeployments []astro.Deployment, deploymentName string) astro.Deployment {
+func deploymentFromName(existingDeployments []astroplatformcore.Deployment, deploymentName string) astroplatformcore.Deployment {
 	for i := range existingDeployments {
-		if existingDeployments[i].Label == deploymentName {
+		if existingDeployments[i].Name == deploymentName {
 			// deployment that matched name
 			return existingDeployments[i]
 		}
 	}
-	return astro.Deployment{}
+	return astroplatformcore.Deployment{}
 }
 
 // getClusterInfoFromName takes clusterName and org as its arguments.
 // It returns the clusterID and list of nodepools if the cluster is found in the organization.
 // It returns an errClusterNotFound if the cluster does not exist in the organization.
-func getClusterInfoFromName(clusterName, org string, coreClient astrocore.CoreClient) (string, []astrocore.NodePool, error) {
+func getClusterInfoFromName(clusterName, organizationShortName string, platformCoreClient astroplatformcore.CoreClient) (string, []astroplatformcore.NodePool, error) {
```

Comment:
if this is using platform core client, we should change this to use orgID instead